Makefile

```diff
@@ -43,7 +43,7 @@ prepare-build-image: build
 	cp broker build/broker
 
 build-image: prepare-build-image
-	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	docker build -f ${BUILD_DIR}/Dockerfile-src -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR}
```

Comment:
Is the release-image build line also wrong?
docker build ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}

Should it be:
docker build -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR} 

If so can that get fixed as well?